osu.Game/Localisation/UserInterfaceStrings.cs

```diff
@@ -0,0 +1,114 @@
+
+        /// <summary>
+        /// "General"
+        /// </summary>
+        public static LocalisableString GeneralHeader => new TranslatableString(getKey(@"general_header"), @"General");
+
+        /// <summary>
+        /// "Rotate cursor when dragging"
+        /// </summary>
+        public static LocalisableString CursorRotation => new TranslatableString(getKey(@"cursor_rotation"), @"Rotate cursor when dragging");
+
+        /// <summary>
+        /// "Menu cursor size"
+        /// </summary>
+        public static LocalisableString MenuCursorSize => new TranslatableString(getKey(@"menu_cursor_size"), @"Menu cursor size");
+
+        /// <summary>
+        /// "Parallax"
+        /// </summary>
+        public static LocalisableString Parallax => new TranslatableString(getKey(@"parallax"), @"Parallax");
+
+        /// <summary>
+        /// "Hold-to-confirm activation time"
+        /// </summary>
+        public static LocalisableString HoldActivationDelay => new TranslatableString(getKey(@"hold_activation_delay"), @"Hold-to-confirm activation time");
+
+        /// <summary>
+        /// "Main Menu"
+        /// </summary>
+        public static LocalisableString MainMenuHeader => new TranslatableString(getKey(@"main_menu_header"), @"Main Menu");
+
+        /// <summary>
+        /// "Interface voices"
+        /// </summary>
+        public static LocalisableString InterfaceVoices => new TranslatableString(getKey(@"interface_voices"), @"Interface voices");
+
+        /// <summary>
+        /// "osu! music theme"
+        /// </summary>
+        public static LocalisableString OsuMusicTheme => new TranslatableString(getKey(@"osu_music_theme"), @"osu! music theme");
+
+        /// <summary>
+        /// "Intro sequence"
+        /// </summary>
+        public static LocalisableString IntroSequence => new TranslatableString(getKey(@"intro_sequence"), @"Intro sequence");
+
+        /// <summary>
+        /// "Background source"
+        /// </summary>
+        public static LocalisableString BackgroundSource => new TranslatableString(getKey(@"background_source"), @"Background source");
+
+        /// <summary>
+        /// "Seasonal backgrounds"
+        /// </summary>
+        public static LocalisableString SeasonalBackgrounds => new TranslatableString(getKey(@"seasonal_backgrounds"), @"Seasonal backgrounds");
+
+        /// <summary>
+        /// "Changes to this setting will only apply with an active osu!supporter tag."
+        /// </summary>
+        public static LocalisableString NotSupporterNote => new TranslatableString(getKey(@"not_supporter_note"), @"Changes to this setting will only apply with an active osu!supporter tag.");
+
+        /// <summary>
+        /// "Song Select"
+        /// </summary>
+        public static LocalisableString SoneSelectHeader => new TranslatableString(getKey(@"song_select_header"), @"Song Select");
```

Comment:
Typo, I would check again for errors as I may be missing more.

```suggestion
        public static LocalisableString SongSelectHeader => new TranslatableString(getKey(@"song_select_header"), @"Song Select");
```